### PR TITLE
added profiles and yaml

### PR DESCRIPTION
**Pull request by yermaalexx**
removed application.properties file;
created yaml files;
created two profiles: h2 and postgre

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+server:
+  port: 8080
+
+minimumAgeOfEmployee: 18
+
+spring:
```
Comment (yermakovsa) [inline src/main/resources/application.yaml:6]: Please don't define profile in application.yaml. You need to set it as env variable. Please look at this link:
https://stackoverflow.com/questions/38520638/how-to-set-spring-profile-from-system-variable
-Dspring.profiles.active=h2 - example to use in VM options

**Comment by yermaalexx** [comment]
removed active profile from application.yaml

File: src/main/resources/application-postgre.yaml
Comment (yermakovsa) [inline src/main/resources/application-postgre.yaml:1]: Please rename it to application-postgre-local.yaml

File: src/main/resources/application-postgre.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+server:
```
Comment (yermakovsa) [inline src/main/resources/application-postgre.yaml]: You have this property in application.yaml. Spring always takes properties from application.yaml and then it looks at active profile. So you don't need to duplicate it in application-h2 and appliaction-postgre

File: .idea/compiler.xml
Comment (yermakovsa) [inline .idea/compiler.xml:1]: Please remove all .idea files from repository

Comment (yermaalexx) [reply .idea/compiler.xml:1]: I will fix it in another pull request (Fix gitignore)

File: src/main/resources/application-postgre.yaml
Comment (yermaalexx) [reply src/main/resources/application-postgre.yaml:1]: ok, I will

File: src/main/resources/application-postgre.yaml (hunk 2 above)
Comment (yermaalexx) [reply src/main/resources/application-postgre.yaml]: I understand, thank you

**Review by yermakovsa — APPROVED** [review]

**Pull request merged**